share/translations/keepassxc_en.ts

```diff
@@ -1779,6 +1779,20 @@ Are you sure you want to continue without a password?</source>
         <source>Failed to change database credentials</source>
         <translation type="unfinished"></translation>
     </message>
+    <message>
+        <source>Weak password</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>You must enter a stronger password to protect your database.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>WARNING! You have set a weak password. If you do not choose a stronger and more complex password, your database may be compromised more easily.
```

Comment:
Can we make this shorter with less technical jargon, please?

Reply:
What about: "WARNING! Using a weak password may expose your accounts to security risks. Do you wish to continue?"?

Reply:
Still way too much tech babble. Just go with

> This is a weak password! For better protection of your secrets, you should choose a stronger password.
> 
> [ OK ] [ Continue with weak password ]

Reply:
This seems ok to me. 

The only doubt I have is about how to implement the "Continue with weak password" button. I see two alternatives:

- I implement the whole warning dialogue as `QMessageBox` so I can add buttons with custom captions, but we would lose the ability of automating GUI testing (at least, I didn't find a way to do that previously)
- I keep using the custom `MessageBox` class for the warning dialogue so the GUI tests can still be automated, but I would have to implement a new button type with the caption above in https://github.com/keepassxreboot/keepassxc/blob/1919c23c097fd5f3e64ae6488aaaabd502db6afb/src/gui/MessageBox.cpp#L59

What would be best? Or is there something am I missing?

Reply:
We use custom buttons everwhere. There are examples in the code. Generic button values are a UX antipattern.

Reply:
Alright. I implemented the custom button and did a rebase.